pkg/operators/v1/olmconfig_types.go

```diff
@@ -0,0 +1,58 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OLMConfigSpec is the spec for an OLMConfig resource.
+type OLMConfigSpec struct {
+	Features Features `json:"features,omitempty"`
+}
+
+// Features contains the list of configurable OLM features.
+type Features struct {
+	// +kubebuilder:default=false
+
+	// DisableCopiedCSVs is used to disable OLM's "Copied CSV" feature
+	// for operators installed at the cluster scope, where a cluster
+	// scoped operator is one that has been installed in an
+	// OperatorGroup that targets all namespaces.
+	// When reenabled, OLM will recreate the "Copied CSVs" for each
+	// cluster scoped operator.
+	DisableCopiedCSVs bool `json:"disableCopiedCSVs,omitempty"`
+}
+
+// OLMConfigStatus is the status for an OLMConfig resource.
+type OLMConfigStatus struct {
+	Conditions []metav1.Condition `json:"conditions,omitempty"`
```

Comment:
Does this need to be `//+optional`?

Reply:
The CRD Generation tool infers that it is options by the `omitempty` tag.